src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+@import './variables';
+@import './calendar';
+
 body {
```

Comment:
you could also add another css for more global code and leave index.scss just for imports